controls/cis_rhel8.yml

```diff
@@ -0,0 +1,2285 @@
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - dconf_gnome_banner_enabled
+      - dconf_gnome_login_banner_text
+      - login_banner_text=usgcb_default
+
+  - id: 1.9
+    title: Ensure updates, patches, and additional security software are installed (Manual)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+    related_rules:
+      - security_patches_up_to_date
+
+  - id: 1.10
+    title: Ensure system-wide crypto policy is not legacy (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - configure_crypto_policy
+
+  # This rule works in conjunction with the configure_crypto_policy above.
+  # If a system is remediated to CIS Level 1, just the rule above will apply
+  # and will enforce the default value for var_system_crypto_policy (DEFAULT).
+  # If the system is remediated to Level 2 then this rule will be selected,
+  # and the value applied by the rule above will will be overridden to
+  # FUTURE through the var_system_crypto_policy variable.
+  - id: 1.11
+    title: Ensure system-wide crypto policy is FUTURE or FIPS (Automated)
+    levels:
+      - l2_server
+      - l2_workstation
+    automated: yes
+    rules:
+      - var_system_crypto_policy=future
+
+  - id: 2.1.1
+    title: Ensure xinetd is not installed (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - package_xinetd_removed
+
+  - id: 2.2.1.1
+    title: Ensure time synchronization is in use (Manual)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+    related_rules:
+      - package_chrony_installed
+
+  - id: 2.1.1
+    title: Ensure chrony is configured (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - service_chronyd_enabled
+      - chronyd_specify_remote_server
+      - chronyd_run_as_chrony_user
+
+  - id: 2.2.2
+    title: Ensure chrony is configured (Automated)
+    levels:
+      - l1_server
+    automated: yes
+    rules:
+      - package_xorg-x11-server-common_removed
+      - xwindows_runlevel_target
+
+  - id: 2.2.3
+    title: Ensure rsync service is not enabled (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - service_rsyncd_disabled
+
+  - id: 2.2.4
+    title: Ensure Avahi Server is not enabled (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - service_avahi-daemon_disabled
+
+  - id: 2.2.5
+    title: Ensure SNMP Server is not enabled (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - service_snmpd_disabled
+
+  - id: 2.2.6
+    title: Ensure HTTP Proxy Server is not enabled (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - package_squid_removed
```

Comment:
I think service_squid_disabled is better aligned with the benchmark.

Reply:
That seems fair - fixed.